MozillaFoundation/foundation.mozilla.org
# Replaced newsletter signup form in PNI product creep vote section with the new implementation approach

Replaced newsletter signup form in PNI product creep vote section with the new implementation approach.

Related to #11787 

<img width="755" alt="image" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/2896608/c5da194a-6672-47d1-ae42-9c545dba895b">


### To test

1. Go to any PNI product page on review app. e.g., https://foundation-s-11787-pni--jl3fjv.herokuapp.com/en/privacynotincluded/green-above-meeting-lay-blue/
2. Go to the voting section (titled "How creepy do you think this is?") to vote
3. The newsletter signup screen shows up only on the 2nd or 3rd time you voted on the same product page. If you wanna use the same product page to test, a way to hack it is to change sessionStorag's voteCount to 2. i.e., run `sessionStorage.setItem("voteCount", 2)` on console, refresh the page, and vote again
4. Test to see if newsletter signup can go through and if the "No thanks" button works as expected.
